cmd/cmd.go

```diff
@@ -194,6 +196,11 @@ func commonConfigure() error {
 		return fmt.Errorf("could not parse configuration: %s", err)
 	}
 
+	// If list of federations is specified on the CLI, mutate the configuration directly
+	if len(federations) > 0 {
```

Comment:
@ripienaar what I thought I might have to do to support overriding the federations at the cli, otherwise I can create a method in Choria to set the federations